lib/build/bundle.js

```diff
@@ -0,0 +1,63 @@
+var assign = require("lodash").assign;
+var minify = require("../stream/minify");
+var devBundle = require("../stream/dev_bundle");
```

Comment:
the filtered stream would be empty since `minimatch(anything, "");` always returns false.... Instead of using an empty string as default, I'll use the match anything blog pattern ("**") and that would indeed create a bundle of the entire graph.